saleae/saleae.py

```diff
@@ -156,8 +156,8 @@ def launch_logic(timeout=15, quiet=False, logic_path=None, args=None):
 		connection_start = time.time()
 		while time.time() < (connection_start + timeout):
 			with contextlib.closing(socket.socket(socket.AF_INET, socket.SOCK_STREAM)) as sock:
-				if sock.connect_ex(('localhost', 10429)) == 0:
-					print('connection detected after {} seconds'.format(time.time()-connection_start))
+				if sock.connect_ex((host, port)) == 0:
```

Comment:
@ppannuto I didn't check this if values are None since I'm assuming that it's always going to receive these two values.